src/scss/custom/_footer.scss

```diff
@@ -6,11 +6,11 @@ div[data-footer-col] {
   width: 20px;
 }
 
-#navlink-footer-id {
+.navlink-footer-id {
```

Comment:
Please consider if it could  be an idea to change the name of navlink-footer-id to something else without the "-id" in it, since it is moved to be a class ??

Reply:
maybe. it is not connected to some modal, I think it is just for CSS styling.